index.js

```diff
@@ -21,15 +21,10 @@ fontmachine.makeGlyphs = function(opts, callback) {
     var stack = [];
     fontnik.load(opts.font, function(err, faces) {
         if (err) return callback(err);
-        // temp workaround: fill this out
+
```

Comment:
I never found any fonts that threw this error, but it seems like font files [can support more than one face](http://www.freetype.org/freetype2/docs/glyphs/glyphs-1.html) so it's a good check to have. Do any of the fontnik fixtures have multiple faces?